site/topics/logic/combinatorial-logic.rst

```diff
@@ -2,6 +2,29 @@
 Combinatorial Logic
```

Comment:
I dislike that this name is common. _Combinatorial_, mathematically, specifically means to disregard arrangement. Logic gates and circuits definitely care about the arrangement. They are, however, _combinators_ -- functions with no free variables, ie they are defined entirely by their inputs and outputs.

English am I right.

Reply:
It's actually a typo from me --- it should be combinational